device/transport/http/src/http_errors.ts

```diff
@@ -16,6 +19,9 @@ export class HttpTransportError extends Error {
  * - the body of  the HTTP response, containing the explanation of why the request failed
  * - the HTTP response object itself]
  */
+/**
+ * @private
```

Comment:
Seems odd.

Reply:
the `@private` comment just removes this from the web page. since this is not used by SDK users, no need to have it in the docs.